src/js/ifs/worker.js

```diff
@@ -55,17 +55,17 @@ function init(d) {
     generation = d.generation;
     width = d.width;
     height = d.height;
-    var numIFS = d.systems.length;
-    var numTrafos = 0;
-    var i, j;
+    const numIFS = d.systems.length;
+    let numTrafos = 0;
+    let i, j;
```

Comment:
Remove `let i, j;` and instead declare i and j in the 3 for statements where they are used.